README.md

```diff
@@ -58,6 +60,15 @@ We use --recursive flag for our dependency on googletest framework.
 
 First create a build directory from the root of the project. In the build directory, run the following commands depending on your environment.
 
+For compiling with SSL support: 
+- You need have the openssl installed in your development environment. Furthermore, the openssl include directory should be added to include directories and the openssl library directory should be listed in the link directories. Furthermore you need to set the openssl libraries to link.
```

Comment:
To compile with SSL support:
- You need to have the openssl installed in your development environment: (i) Add openssl `include` directory to include directories, (ii) Add openssl `library` directory to the link directories list, (iii) Set the openssl libraries to link.